cmd/thor/flags.go

```diff
@@ -142,4 +142,13 @@ var (
 		Value: 16,
 		Usage: "set tx limit per account in pool",
 	}
+	telemetryEnabledFlag = cli.BoolTFlag{
+		Name:  "telemetry-enabled",
+		Usage: "enables telemetry server",
+	}
+	telemetryAddrFlag = cli.StringFlag{
+		Name:  "telemetry-addr",
+		Value: "127.0.0.1:2112",
```

Comment:
`apiAddrFlag` defaults `localhost:8669`, should we change one of them for consistency?

Reply:
```suggestion
		Value: "localhost:2112",
```